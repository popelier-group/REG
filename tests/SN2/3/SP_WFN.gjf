# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.42084800   -0.00007500    0.00017900
 H                 -0.72848800   -0.87953600   -0.54734000
 H                 -0.72855700    0.91381000   -0.48767100
 H                 -0.72837900   -0.03452900    1.03560700
 Br                 1.61574500   -0.00000300    0.00001800
 Cl                -3.43903800    0.00004000   -0.00011900

frame.wfn
