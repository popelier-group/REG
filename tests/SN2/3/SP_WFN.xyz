
6

C       -0.309694    0.005213    0.039712
H       -0.512086   -0.539080    0.951114
H       -0.738802   -0.475727   -0.827716
H       -0.593481    1.045395    0.112919
Br        1.710145   -0.027808   -0.218884
Cl       -3.303091    0.053613    0.422727
