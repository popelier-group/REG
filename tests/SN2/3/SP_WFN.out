 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.3/Gau-70828.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.3/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     70829.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.42085  -0.00008   0.00018 
 H                    -0.72849  -0.87954  -0.54734 
 H                    -0.72856   0.91381  -0.48767 
 H                    -0.72838  -0.03453   1.03561 
 Br                    1.61575  0.         0.00002 
 Cl                   -3.43904   0.00004  -0.00012 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.420848   -0.000075    0.000179
      2          1           0       -0.728488   -0.879536   -0.547340
      3          1           0       -0.728557    0.913810   -0.487671
      4          1           0       -0.728379   -0.034529    1.035607
      5         35           0        1.615745   -0.000003    0.000018
      6         17           0       -3.439038    0.000040   -0.000119
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.080681   0.000000
     3  H    1.080680   1.794338   0.000000
     4  H    1.080682   1.794368   1.794359   0.000000
     5  Br   2.036593   2.562929   2.562937   2.562919   0.000000
     6  Cl   3.018190   2.901756   2.901618   2.901999   5.054783
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.309694    0.005213    0.039712
      2          1           0       -0.512086   -0.539080    0.951114
      3          1           0       -0.738802   -0.475727   -0.827716
      4          1           0       -0.593481    1.045395    0.112919
      5         35           0        1.710145   -0.027808   -0.218884
      6         17           0       -3.303091    0.053613    0.422727
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.7453730           0.8086646           0.8086646
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       175.4174609609 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.44D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39473323     A.U. after   12 cycles
            NFock= 12  Conv=0.12D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.68856-101.25755 -62.34158 -56.16143 -56.15825
 Alpha  occ. eigenvalues --  -56.15825 -10.08097  -9.17060  -8.55095  -6.92926
 Alpha  occ. eigenvalues --   -6.92870  -6.92870  -6.38071  -6.37021  -6.37021
 Alpha  occ. eigenvalues --   -2.49187  -2.48889  -2.48889  -2.48056  -2.48056
 Alpha  occ. eigenvalues --   -0.66145  -0.53946  -0.50943  -0.30139  -0.30138
 Alpha  occ. eigenvalues --   -0.24252  -0.15666  -0.15666  -0.05848  -0.05712
 Alpha  occ. eigenvalues --   -0.05712
 Alpha virt. eigenvalues --    0.09756   0.10902   0.14032   0.14033   0.16101
 Alpha virt. eigenvalues --    0.17410   0.18023   0.18023   0.21504   0.21878
 Alpha virt. eigenvalues --    0.21879   0.24315   0.24315   0.24579   0.30952
 Alpha virt. eigenvalues --    0.30952   0.31566   0.34086   0.40529   0.50199
 Alpha virt. eigenvalues --    0.52240   0.52240   0.53981   0.53981   0.66463
 Alpha virt. eigenvalues --    0.67759   0.67759   0.70000   0.74697   0.74697
 Alpha virt. eigenvalues --    0.75681   0.77209   0.77209   0.82667   0.82667
 Alpha virt. eigenvalues --    0.84231   0.95423   1.08061   1.14851   1.14851
 Alpha virt. eigenvalues --    1.15273   1.15273   1.16449   1.22979   1.22980
 Alpha virt. eigenvalues --    1.24008   1.56533   1.67402   1.67403   1.80671
 Alpha virt. eigenvalues --    1.84565   1.84565   2.08332   2.12467   2.12467
 Alpha virt. eigenvalues --    2.28867   2.30962   2.30963   2.52107   2.52107
 Alpha virt. eigenvalues --    2.55742   2.59992   2.64480   2.64480   2.68268
 Alpha virt. eigenvalues --    2.68268   3.11829   3.13939   3.85737   3.85738
 Alpha virt. eigenvalues --    4.27829   4.27829   4.37025   4.37025   4.65715
 Alpha virt. eigenvalues --    6.73227   7.69234   7.69234   7.79494   9.87439
 Alpha virt. eigenvalues --   24.04161  26.07973  26.07974  26.19443  48.02578
 Alpha virt. eigenvalues --  215.83902 289.89212 289.89212 290.012861020.80344
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.061347   0.410505   0.410516   0.410484   0.098431  -0.051294
     2  H    0.410505   0.528182  -0.035871  -0.035858  -0.054615   0.008467
     3  H    0.410516  -0.035871   0.528160  -0.035862  -0.054611   0.008467
     4  H    0.410484  -0.035858  -0.035862   0.528222  -0.054620   0.008467
     5  Br   0.098431  -0.054615  -0.054611  -0.054620  35.361555   0.007788
     6  Cl  -0.051294   0.008467   0.008467   0.008467   0.007788  17.911743
 Mulliken charges:
               1
     1  C   -0.339989
     2  H    0.179189
     3  H    0.179201
     4  H    0.179167
     5  Br  -0.303928
     6  Cl  -0.893639
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.197567
     5  Br  -0.303928
     6  Cl  -0.893639
 Electronic spatial extent (au):  <R**2>=           1200.8113
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             10.8266    Y=             -0.1755    Z=             -1.3855  Tot=             10.9163
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -97.2730   YY=            -43.8152   ZZ=            -44.6770
   XY=              0.8679   XZ=              6.8433   YZ=             -0.1111
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -35.3513   YY=             18.1065   ZZ=             17.2448
   XY=              0.8679   XZ=              6.8433   YZ=             -0.1111
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            231.7452  YYY=             -0.2619  ZZZ=             -9.3449  XYY=             23.3011
  XXY=             -3.0222  XXZ=            -23.7177  XZZ=             25.8619  YZZ=             -1.2777
  YYZ=             -2.8888  XYZ=              0.2285
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -2047.7863 YYYY=            -70.4661 ZZZZ=            -94.4577 XXXY=             20.1431
 XXXZ=            158.6141 YYYX=             11.4985 YYYZ=             -1.4689 ZZZX=             95.5940
 ZZZY=             -1.7402 XXYY=           -269.7259 XXZZ=           -281.7072 YYZZ=            -27.4715
 XXYZ=             -1.4296 YYXZ=             31.4658 ZZXY=              4.6023
 N-N= 1.754174609609D+02 E-N=-7.702129407327D+03  KE= 3.069995382384D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE333\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -0.420848,-0.000075,0.000179\H,0,-0.728488,-0.879536,-0.54734\H,0,-0.7
 28557,0.91381,-0.487671\H,0,-0.728379,-0.034529,1.035607\Br,0,1.615745
 ,-0.000003,0.000018\Cl,0,-3.439038,0.00004,-0.000119\\Version=ES64L-G1
 6RevC.01\State=1-A\HF=-3074.3947332\RMSD=1.199e-09\Dipole=4.2948134,-0
 .0000961,0.0002629\Quadrupole=-26.944422,13.4721921,13.47223,0.0004997
 ,-0.0014873,0.000005\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 KNOWLEDGE IS OF TWO KINDS:  WE KNOW A SUBJECT OURSELVES OR WE KNOW 
 WHERE WE CAN FIND INFORMATION UPON IT.

                                    -- SAMUEL JOHNSON
 Job cpu time:       0 days  0 hours  0 minutes 29.1 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:08 2020.
